Integrations/integration-McAfeeWebGateway.yml

```diff
@@ -0,0 +1,328 @@
+description: Blacklist/Whitelist handling
+configuration:
+- display: 'URL (e.g. https://192.168.100.55:12345) '
```

Comment:
1. we usually split port and url
2. use our standard display for url - Server URL (e.g. https://192.168.0.1)